pages/linux/aspell.md

```diff
@@ -0,0 +1,23 @@
+# aspell
+
+> Interactive spell checker.
+
+- Spell check a single file:
+
+`aspell check file`
+
+- List mispelled words from standard input:
+
+`cat file | aspell list`
+
+- Show available dictionary files:
+
+`aspell dicts`
+
+- Run aspell with different language (takes two letter ISO 639 language code):
+
+`aspell --lang=lang-code`
```

Comment:
Could `lang-code` be surrounded in `{{` and `}}` please?

Reply:
done